Extensions/postarchive.js

```diff
@@ -244,31 +265,31 @@ XKit.extensions.postarchive = {
 
 				} else {
 
-					m_post_list_html = m_post_list_html + "<div data-category=\"" + XKit.extensions.postarchive.categories[i].id + "\" class=\"xkit-postarchive-no-posts-index " + m_collapse_class_post + "\">No posts</div>";
+					m_post_list_html = `${m_post_list_html}<div data-category="${XKit.extensions.postarchive.categories[i].id}" class="xkit-postarchive-no-posts-index ${m_collapse_class_post}">No posts</div>`;
 
 				}
 
 			}
 
 		}
 
-		var m_html = "<div id=\"xkit-postarchive-background\">&nbsp;</div>" +
+		var m_html = `${"<div id=\"xkit-postarchive-background\">&nbsp;</div>" +
 				"<div id=\"xkit-postarchive-container\">" +
 					"<div id=\"xkit-postarchive-sidebar\" class=\"nano\">" +
-						"<div id=\"xkit-postarchive-sidebar-content\" class=\"content\">" +
-							m_post_list_html +
-						"</div>" +
-					"</div>" +
-					"<input type=\"text\" placeholder=\"Search archived post titles..\" id=\"xkit-postarchive-search\">" +
-					"<div id=\"xkit-postarchive-export\" class=\"xkit-button\">Export</div>" +
-					"<div id=\"xkit-postarchive-import\" class=\"xkit-button\">Import</div>" +
-					"<div id=\"xkit-postarchive-content\" class=\"xkit-postarchive-no-post-selected\">" +
-						"Select a post from the right to get started." +
-					"</div>" +
-					"<div id=\"xkit-postarchive-remove-this\" class=\"xkit-button hidden\">remove</div>" +
-					"<div id=\"xkit-postarchive-recategorize-this\" class=\"xkit-button hidden\">recategorize</div>" +
-					"<div id=\"xkit-postarchive-rename-this\" class=\"xkit-button hidden\">rename</div>" +
-				"</div>";
+						"<div id=\"xkit-postarchive-sidebar-content\" class=\"content\">"}${
```

Comment:
Why does this section still use concatenation in addition to the template string?